Invoro.WebApp/Invoro/invoro/src/lib/components/FeaturesTable/FeaturesTable.component.tsx

```diff
@@ -98,4 +100,12 @@ export default class FeaturesTableComponent extends React.Component<FeaturesTabl
       return <TableCell align="left"></TableCell>
     }
   }
+
+  private getFeaturesCategoryComparerFunc(): (a: FeaturesCategory, b: FeaturesCategory) => number {
+    return this.props.featuresCategoryCompareFn ? this.props.featuresCategoryCompareFn : FeaturesCategory.Compare;
+  }
+
+  private getFeaturesComparerFunc(): (a: Feature, b: Feature) => number {
```

Comment:
There is a way to pass default values to properties. look here: https://stackoverflow.com/questions/37282159/default-property-value-in-react-component-using-typescript